meson.build

```diff
@@ -0,0 +1,171 @@
+    'Quake/crc.c',
+    'Quake/cvar.c',
+    'Quake/gl_draw.c',
+    'Quake/gl_fog.c',
+    'Quake/gl_heap.c',
+    'Quake/gl_mesh.c',
+    'Quake/gl_model.c',
+    'Quake/gl_refrag.c',
+    'Quake/gl_rlight.c',
+    'Quake/gl_rmain.c',
+    'Quake/gl_rmisc.c',
+    'Quake/gl_screen.c',
+    'Quake/gl_sky.c',
+    'Quake/gl_texmgr.c',
+    'Quake/gl_vidsdl.c',
+    'Quake/gl_warp.c',
+    'Quake/host.c',
+    'Quake/host_cmd.c',
+    'Quake/image.c',
+    'Quake/in_sdl.c',
+    'Quake/keys.c',
+    'Quake/main_sdl.c',
+    'Quake/mathlib.c',
+    'Quake/mdfour.c',
+    'Quake/mem.c',
+    'Quake/menu.c',
+    'Quake/miniz.c',
+    'Quake/net_bsd.c',
+    'Quake/net_dgrm.c',
+    'Quake/net_loop.c',
+    'Quake/net_main.c',
+    'Quake/net_udp.c',
+    'Quake/palette.c',
+    'Quake/pl_linux.c',
+    'Quake/pr_cmds.c',
+    'Quake/pr_edict.c',
+    'Quake/pr_exec.c',
+    'Quake/pr_ext.c',
+    'Quake/r_alias.c',
+    'Quake/r_brush.c',
+    'Quake/r_part.c',
+    'Quake/r_part_fte.c',
+    'Quake/r_sprite.c',
+    'Quake/r_world.c',
+    'Quake/sbar.c',
+    'Quake/snd_codec.c',
+    'Quake/snd_dma.c',
+    'Quake/snd_mem.c',
+    'Quake/snd_mix.c',
+    'Quake/snd_sdl.c',
+    'Quake/snd_umx.c',
+    'Quake/snd_wave.c',
+    'Quake/strlcat.c',
+    'Quake/strlcpy.c',
+    'Quake/sv_main.c',
+    'Quake/sv_move.c',
+    'Quake/sv_phys.c',
+    'Quake/sv_user.c',
+    'Quake/sys_sdl_unix.c',
+    'Quake/tasks.c',
+    'Quake/view.c',
+    'Quake/wad.c',
+    'Quake/world.c',
+    'Shaders/Compiled/Release/alias_alphatest.frag.c',
+    'Shaders/Compiled/Release/alias.frag.c',
+    'Shaders/Compiled/Release/alias.vert.c',
+    'Shaders/Compiled/Release/basic_alphatest.frag.c',
+    'Shaders/Compiled/Release/basic.frag.c',
+    'Shaders/Compiled/Release/basic_notex.frag.c',
+    'Shaders/Compiled/Release/basic.vert.c',
+    'Shaders/Compiled/Release/cs_tex_warp.comp.c',
+    'Shaders/Compiled/Release/postprocess.frag.c',
+    'Shaders/Compiled/Release/postprocess.vert.c',
+    'Shaders/Compiled/Release/screen_effects_10bit.comp.c',
+    'Shaders/Compiled/Release/screen_effects_10bit_scale.comp.c',
+    'Shaders/Compiled/Release/screen_effects_10bit_scale_sops.comp.c',
+    'Shaders/Compiled/Release/screen_effects_8bit.comp.c',
+    'Shaders/Compiled/Release/screen_effects_8bit_scale.comp.c',
+    'Shaders/Compiled/Release/screen_effects_8bit_scale_sops.comp.c',
+    'Shaders/Compiled/Release/showtris.frag.c',
+    'Shaders/Compiled/Release/showtris.vert.c',
+    'Shaders/Compiled/Release/sky_box.frag.c',
+    'Shaders/Compiled/Release/sky_layer.frag.c',
+    'Shaders/Compiled/Release/sky_layer.vert.c',
+    'Shaders/Compiled/Release/update_lightmap.comp.c',
+    'Shaders/Compiled/Release/world.frag.c',
+    'Shaders/Compiled/Release/world.vert.c',
+]
+
+cc = meson.get_compiler('c')
+deps = [
+    cc.find_library('m', required : false),
+    cc.find_library('dl', required : false),
+    cc.find_library('pthread', required : false),
```

Comment:
For dl, you can use (since Meson 0.62.0) https://mesonbuild.com/Dependencies.html#dl-libdl
For pthread, it's always advised to use https://mesonbuild.com/Threads.html

Reply:
Thanks, will check. this is my first time using meson

Reply:
My system only has Meson 0.61 so this will have to wait. I changed pthread to dependency('threads') though